backend/services/Auth/LoginHandler.js

```diff
@@ -0,0 +1,22 @@
+class LoginHandler {
+    constructor(userService, bcryptjs) {
+        this.userService = userService;
```

Comment:
this need to be repository (DDD standard)

Reply:
done